core/datainput.gms

```diff
@@ -6,6 +6,11 @@
 *** |  Contact: remind@pik-potsdam.de
 *** SOF ./core/datainput.gms
 
+*** carbon intensities of coal, oil, and gas
+pm_cintraw("pecoal") = 26.1 / s_zj_2_twa;
+pm_cintraw("peoil")  = 20.0 / s_zj_2_twa;
+pm_cintraw("pegas")  = 15.0 / s_zj_2_twa;
+
```

Comment:
Could you put this [here](https://github.com/remindmodel/remind/blob/612254c8f3ff67c48e24775f6aca4d91ab4e89cd/core/datainput.gms#L684) to where other emissions factors are read-in? 

We might take with together with the other emissions factors to the new emissions module. That way, chance is lower that we overlook it.

Reply:
`pm_cintraw` is read before the line you are suggesting --> https://github.com/remindmodel/remind/blob/612254c8f3ff67c48e24775f6aca4d91ab4e89cd/core/datainput.gms#L684

Reply:
Ok. I'd say just put it before that. And add comment "emissions factor of primary energy fossil fuels". It is all not very well sorted but I hope with moving to a new emissions module we can improve that.